castero/queue.py

```diff
@@ -169,6 +180,14 @@ def _sanitize_volume(self) -> None:
         elif self._volume < 0:
             self._volume = 0
 
+    def _sanitize_speed(self) -> None:
+        """Ensure the speed is an acceptable value (0.5-2.0 inclusive)
+        """
+        if (self._speed < 0.5):
```

Comment:
These are fine values, but would prefer to define them as class constants (you could probably add a MAX_VOLUME and MIN_VOLUME from _sanitize_volume too).